src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -162,6 +163,191 @@ def verify_daos_pool_result(self, result, action, expect, err_code):
                 " =Test Passed on verify_daos_pool %s expected error of %s.\n",
                 action, expect)
 
+    def verify_cont_rw_attribute(self, action, expect, attribute, value=None):
+        """verify container rw attribute.
+
+        Args:
+            action (str): daos pool read or write.
+            expect (str): expecting pass or deny.
+            attribute (str): Container attribute to be verified.
+            value (str optional): Container attribute value to write.
+
+        """
+        if action.lower() == "write":
+            result = self.set_container_attribute(self.pool_uuid,
+                                                  self.pool_svc,
+                                                  self.container_uuid,
+                                                  attribute,
+                                                  value)
+        elif action.lower() == "read":
+            result = self.get_container_attribute(self.pool_uuid,
+                                                  self.pool_svc,
+                                                  self.container_uuid,
+                                                  attribute)
+        else:
+            self.fail(
+                "##In verify_cont_rw_attribute, "
+                "invalid action: {}".format(action))
+        self.log.info(
+            "  In verify_cont_rw_attribute %s.\n =daos_cmd.run() result:\n%s",
+            action, result)
+        self.verify_daos_pool_cont_result(result, action, expect, DENY_ACCESS)
+
+    def verify_cont_rw_property(
+        self, action, expect, cont_property=None, value=None):
+        """verify container rw property.
+
+        Args:
+            action (str): daos container read or write.
+            expect (str): expecting pass or deny.
+            cont_property (str optional): Container property to be verified.
+            value (str optional): Container property value to write.
+
+        """
+        if action.lower() == "write":
+            result = self.set_container_property(self.pool_uuid,
+                                                 self.pool_svc,
+                                                 self.container_uuid,
```

Comment:
(pylint-no-member) Instance of 'PoolSecurityTestBase' has no 'container_uuid' member